### PR TITLE
Tag list branch

### PR DESCRIPTION
**Pull request by MaeYoungPhan**
Closes Ticket # 21

# Description of Proposed Changes

When user selects the "Tag Management" from the navbar, they are directed to /tags, and see a list of all tags
each tag has an edit and delete button (functionality for buttons come later)
List is sorted in alphabetical order

# Steps to Test

1. Fetch the [`tag-list-branch`](https://github.com/nss-day-cohort-60/rare-client-tenacious-tators/compare/main...tag-list-branch?expand=1) branch and switch to it
2. npm start to localhost 
3. login
4. Click "tag manager" in upper navigation
5. Expectation : you are redirected to "/tags" and an alphabetized* list of all tags displays on the page. Each tag should have an edit and delete button (non-functional.)
*Assumption: Tags will only be alphabetized once gets all tags by labelASC is merged with server

### CODE REVIEW

File: src/views/ApplicationViews.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Posts } from "../components/posts/Posts"
 import { MyPosts } from "../components/posts/MyPosts"
 import { Categories } from "../components/categories/Categories"
 import { Tags } from "../components/tags/Tag"
```
Comment (lauren-hanson) [inline src/views/ApplicationViews.js:8]: I'm receiving an error because the import. Could you update the name for line 8 & line 27?

**Review by lauren-hanson — CHANGES_REQUESTED** [review]
One edit to make!

File: src/views/ApplicationViews.js (hunk 1 above)
Comment (MaeYoungPhan) [reply src/views/ApplicationViews.js:8]: I must have accidentally pushed a former version. I'll go fix that.

**Review by lauren-hanson — APPROVED** [review]

**Comment by lauren-hanson** [comment]
You are all good to go! Thank you for your patience on this one.

**Pull request merged**